### DISCUSSION TITLE
Uninstaller doesn't seem to work

### DISCUSSION THREAD

**Discussion by tomwolsky** [latenitefilms/BRAWToolbox#30] [Support] [answered]
It runs and I get a narrow window with a long string of text. The window doesn't scroll so I can;'t get to the bottom of it. Have to force quit. The extension is still installed.

**Comment by latenitefilms** [comment]
Can you share a screenshot?

**Comment by tomwolsky** [comment]
Looks like this
<img width="907" alt="60%_Screenshot 2023-01-25 at 13 56 06" src="https://user-images.githubusercontent.com/18217685/214657331-f270b427-38c5-48ef-95c0-9023c159369b.png">

**Reply by latenitefilms** [reply]
Ah ok, thanks for reporting! It must be some extra macOS Ventura security permissions - looks like the uninstall tool can't delete things, even though it's not a sandboxed app. I'll go back to the drawing board.

**Reply by latenitefilms** [reply]
Added to issue #42.

**Comment by latenitefilms** [comment, marked as answer]
The Uninstall Tool was updated on Feb 13th.

https://brawtoolbox.io/uninstall/
